packages/Icon/src/index.js

```diff
@@ -0,0 +1,38 @@
+export { default as Add } from "./Add";
+export { default as ArrowDown } from "./ArrowDown";
```

Comment:
is this file created programmatically? 🤔 or do we have to manually add a new icon here if we want to export it?

Reply:
Manually 😂  just took me 3 mins.. we can add a script in the future to generate this file for sure